core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -36,43 +37,68 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
               }
             }
         } ~
-          pathPrefix("billingAccount") {
-            pathEnd {
-              put {
-                entity(as[UpdateRawlsBillingAccountRequest]) { updateProjectRequest =>
-                  complete( userServiceConstructor(userInfo).updateBillingAccount(RawlsBillingProjectName(projectId), updateProjectRequest) )
+        pathPrefix("spendReportConfiguration") {
```

Comment:
This file shows more changes than there really were. The only thing that changed is the name of the function in `UserService` to update a billing account was changed from `updateBillingAccount` to `updateBillingProjectBillingAccount`.